src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -43,125 +43,126 @@ concurrent requests, request rate).
 
 GenAI-Perf generates the specified load, measures the performance of the
 inference server and reports the metrics in a simple table as console output.
-The tool also logs all results in a csv file that can be used to derive
+The tool also logs all results in a csv and json file that can be used to derive
 additional metrics and visualizations. The inference server must already be
 running when GenAI-Perf is run.
 
+You can use GenAI-Perf to run performance benchmarks on
+- [Large Language Models](docs/tutorial.md)
+- [Multi-Modal Models](docs/multi_modal.md)
+- [Embedding Models](docs/embeddings.md)
+- [Ranking Models](docs/rankings.md)
+- [Multiple LoRA Adapters](docs/lora.md)
+
 > [!Note]
 > GenAI-Perf is currently in early release and under rapid development. While we
 > will try to remain consistent, command line options and functionality are
 > subject to change as the tool matures.
 
-# Installation
+</br>
 
-## Triton SDK Container
+<!--
+======================
+INSTALLATION
+======================
+-->
 
-Available starting with the 24.03 release of the
-[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
+## Installation
 
-Run the Triton Inference Server SDK docker container:
+The easiest way to install GenAI-Perf is through
+[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
+Install the latest release using the following command:
 
 ```bash
-export RELEASE="yy.mm" # e.g. export RELEASE="24.03"
+export RELEASE="yy.mm" # e.g. export RELEASE="24.06"
 
 docker run -it --net=host --gpus=all  nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
+
+# Check out genai_perf command inside the container:
+genai-perf --help
 ```
 
 <details>
 
 <summary>Alternatively, to install from source:</summary>
 
-## From Source
-
-GenAI-Perf depends on Perf Analyzer. Here is how to install Perf Analyzer:
+Since GenAI-Perf depends on Perf Analyzer,
+you'll need to install Perf Analyzer binary:
 
 ### Install Perf Analyzer (Ubuntu, Python 3.8+)
 
-Note: you must already have CUDA 12 installed.
+**NOTE**: you must already have CUDA 12 installed.
```

Comment:
should we add instructions to install cuda if its necessary?

Reply:
I didnt mean explicit instructions but a pointer.
something like this: https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html

Reply:
not a requirement so resolving this comment.

Reply:
ah I see. I thought you meant the full instruction. Adding a link sounds good. I've added the link in the doc.